clusters/app.ci/supplemental-ci-images/baremetal-qe/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- bmanzari
```

Comment:
Hi Biagio, also add `pamoedom` as approver at least to have a third person.

Reply:
Hi, added